lib/active_shipping/rate_estimate.rb

```diff
@@ -95,6 +101,7 @@ def initialize(origin, destination, carrier, service_name, options = {})
       self.delivery_date = @delivery_range.last
       self.insurance_price = options[:insurance_price]
       self.delivery_category = options[:delivery_category]
+      self.shipment_options = options[:shipment_options]
```

Comment:
Want to default this to be an empty array? 

It's the difference between having to `if rate_estimate.shipment_options` guard iteration and things.